tests/graphql/errors/test-call-operator.graphql

```diff
@@ -0,0 +1,32 @@
+#> server-sdl
+schema @server @upstream(baseURL: "http://localhost:3000") {
+  query: Query
+}
+
+type Query {
+  posts: [Post] @http(path: "/posts")
+  userWithoutResolver(id: Int!): User
+  user(id: Int!): User @http(path: "/users/{{args.id}}")
+}
+
+type User {
+  id: Int!
+}
+
+type Post {
+  userId: Int!
+  withoutResolver: User @call(query: "userWithoutResolver", args: {id: "{{value.userId}}"})
+  multipleResolvers: User @http(path: "/users/{{value.userId}}") @call(query: "user", args: {id: "{{value.userId}}"})
+  withoutOperator: User @call(args: {id: "{{value.userId}}"})
+  # invalidResolver: User @call(query: "userWithGraphQLResolver", args: {id: "{{value.userId}}"})
```

Comment:
Needs tests for argument mismatch.

Reply:
Can I rely on `Mustache::parse` to check it?